code/modules/projectiles/guns/projectile.dm

```diff
@@ -6,8 +6,12 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	materials = list(MAT_METAL=1000)
 
+	light_power = 10
+	light_color = LIGHT_COLOR_WHITE //gun flash variables

```

Comment:
What's going on here?

Reply:
Oh woops, code I forgot to take out. This was part of me attempting to put in weapon flashes before I eventually decided to do that in its own PR.